modules/custom_role_iam/README.md

```diff
@@ -0,0 +1,44 @@
+# Module Custom Role IAM
+
+This optional module is used to create custom roles at organization or project level.
+
+## Usage
+
+```hcl
+locals {
+  role_permissions = ["iam.roles.list", "iam.roles.create", "iam.roles.delete"]
+}
+
+module "custom-roles" {
+  source = "../../modules/custom_role_iam/"
+
+  role_level  = "project"
```

Comment:
I think it will be good to showcase both project and org in the readme. WDYT?

Reply:
Added README for org and project custom roles separately.